client/src/utils/unitHelpers.ts

```diff
@@ -18,18 +18,22 @@ export function parseRam(value?: string) {
 export function parseUnitsOfRam(bytes?: number, targetUnit?: string) {
     if (!bytes) return undefined;
 
-    const actualTargetUnit = targetUnit?.replace('b', '');
-    let i = 0;
-    while (((actualTargetUnit === undefined && bytes >= 1000)
-        || (actualTargetUnit !== undefined && UNITS[i] !== actualTargetUnit)) && i < UNITS.length - 1) {
-        i++;
-        bytes /= 1000; // eslint-disable-line no-param-reassign
+    try {
```

Comment:
Thanks for your contribution. Instead of adding a try catch here, we will it may be better to check the null cases like above.